tests/foreman/cli/test_satellitesync.py

```diff
@@ -2059,3 +2059,151 @@ def test_positive_install_package_from_imported_repos(self):
 
         :CaseLevel: System
         """
+
+
+@pytest.fixture(scope='module')
+def module_downstream_sat(module_satellite_host):
+    """Provides Downstream Satellite."""
+    module_satellite_host.cli.Settings.set(
+        {'name': 'subscription_connection_enabled', 'value': 'No'}
+    )
```

Comment:
Do we need an teardown for this to change back to Yes (if default)?

Reply:
No. On the downstream sat we always need this off.